crates/rpc/src/handlers/block.rs

```diff
@@ -208,6 +211,45 @@ pub async fn get_beacon_block_from_id(
         })
 }
 
+pub async fn get_filtered_block_tree(db: &ReamDB) -> Result<Vec<SignedBeaconBlock>, ApiError> {
+    let justified_checkpoint = db
+        .justified_checkpoint_provider()
+        .get()
+        .map_err(|_| ApiError::InternalError)?
+        .ok_or_else(|| ApiError::NotFound(String::from("Justified checkpoint not found")))?;
+    let root = justified_checkpoint.root;
```

Comment:
I need to some more research into if this implementation is correct or not in regards to the endpoint. I don't have time to look into this further right now. I will try to look into it more in a few days when I have some time.